3cn-ecn/nantralPlatform
# Adapt calendar for mobile device

# Description

Adapt to mobile interface.
Review hierarchy of components.
Review code.

# Tests

1. Steps to reproduce :

- Create some events *when you want*.
- Try to do some of them with common time area, with very long duration, and in different weeks.
- Go to the calendar in the event page.
- Change size of screen.

2. Expected results : 

- The calendar and the events adapt their size.
- The display is pretty.

# Screenshots

![image](https://user-images.githubusercontent.com/97662746/218026718-f3bc081e-aac5-4edc-be1f-3b534e55e74a.png)
![image](https://user-images.githubusercontent.com/97662746/218026809-3fdca693-e793-4377-b336-121e8a44ddbd.png)
